src/routes.js

```diff
@@ -8,13 +8,25 @@ import Home from 'views/Home';
 import Info from 'views/Info';
 import NotFound from 'views/NotFound';
 
+function trackLink() {
+  /* global Piwik */
+  try {
+    const tracker = Piwik.getTracker("http://analytics.hel.ninja/piwik/piwik.php", 2);
```

Comment:
The tracking URL should be configurable. After all, this is supposed to be a reusable app...